src/schoology-extractor/schoology_extractor/schoology_extract_facade.py

```diff
@@ -92,8 +91,8 @@ def get_users(self) -> pd.DataFrame:
         roles_df = sync.sync_resource(
             RESOURCE_NAMES.ROLE,
             self._db_engine,
-            roles_list,
-            sync_column_types.ROLE_COLUMN_TYPES_MAPPING)
+            roles_list
+            )

```

Comment:
Now we don't need to map the column types, everything is written as a string in a single column.